public/docs/ts/latest/guide/architecture.jade

```diff
@@ -73,7 +73,7 @@ figure
     Learn more</a> about decorators on the web.
 :marked
   `NgModule` is a decorator function that takes a single metadata object whose properties describe the module. 
-  The most important are
+  The most important are:
   * `declarations` - the _view classes_ that belong to this module.
   Angular has three kinds of view classes: [components](#components), [directives](#directives) and [pipes](pipes.html).
 
```

Comment:
I can't leave a comment on lines you didn't change, but just after this, "subset" (in the `exports` item) seems like it should be "the subset"